packages/cli/src/index.js

```diff
@@ -79,7 +79,7 @@ const loadDotEnvDefaultsMiddleware = () => {
   config({
     path: path.join(getPaths().base, '.env'),
     defaults: path.join(getPaths().base, '.env.defaults'),
-    encoding: 'utf8',
+    multiline: true,
```

Comment:
Partly fixes https://github.com/redwoodjs/redwood/issues/5657

Reply:
Do you think we could add a test for this?

Reply:
Sorry I didn't see this comment; I'll revisit

Reply:
No worries! 👍👍👍